website_sale_close/views/website.xml

```diff
@@ -3,15 +3,37 @@
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 
 <odoo>
-    <record id="website_view_form" model="ir.ui.view">
-        <field name="name">website.form</field>
-        <field name="model">website</field>
-        <field name="inherit_id" ref="website_payment.website_settings_payment"/>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
```

Comment:
Do we need to do something at the migration script level when renaming view ids?

Reply:
Yes, you'r write. I may check that more deeply.

Reply:
You're the migration boss

Reply:
In facts, it's ok, because the value where stored in `website` object and that doesn't change. There is just a new abstraction upon that called `res_config_settings`. So migration, should run smoothly. If not a special script will be written for that case.